package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-sfi-apply-web",
-  "version": "1.39.1",
```

Comment:
Maybe bump the version to `1.40.0` as its a functional change

Reply:
changed